### ISSUE TITLE
navigation pop-up doesn't work (staging)

### ISSUE THREAD

**Issue by graybeal** [metadatacenter/cedar-project#630]
When in staging, any attempt to move or copy a document pops up the navigation window, which can only navigate upwards -- can't navigate down into folders. This is a blocking bug for me to get ready for AIRR, and maybe for Ahmad's demo.

**Comment by graybeal** [comment]
The movie demonstrating this is in box in the folder CEDAR Project Team > CEDAR Internal > Troubleshooting  (https://stanfordmedicine.app.box.com/folder/42677472044).

**Comment by marcosmro** [comment]
It appears that the navigation window is appending the contents of the folder that the user is trying to open to the current list of folders, instead of replacing them. For example, I have the following folder structure:

All / Users / Marcos Martinez Romero / folder1 / folder2 / folder3

I created 'Template1' inside 'folder1' and tried to move it to 'folder3'. 

Initially, the navigation window shows:
![screen shot 2017-12-03 at 11 57 21 am](https://user-images.githubusercontent.com/7634440/33529143-38aa431a-d821-11e7-8968-625d7c4567b8.png)

Now, I click on 'folder 1', and the windows appends the contents to the current contents, instead of replacing them:

![screen shot 2017-12-03 at 11 57 30 am](https://user-images.githubusercontent.com/7634440/33529145-4100613e-d821-11e7-9161-849767f10879.png)

And the same happens when clicking on 'folder2':

![screen shot 2017-12-03 at 11 57 39 am](https://user-images.githubusercontent.com/7634440/33529147-45c382aa-d821-11e7-8bc4-ab8947d0b124.png)

I took a look at the REST calls and everything looks good, so I think that this is a UI issue. @willrett may have some additional thoughts on this.
